client/state/data-layer/wpcom/me/block/sites/delete/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @format
+ */
+
+/**
+ * External Dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import { READER_SITE_UNBLOCK } from 'state/action-types';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { errorNotice, plainNotice } from 'state/notices/actions';
+import { blockSite } from 'state/reader/site-blocks/actions';
+import { bypassDataLayer } from 'state/data-layer/utils';
+
+export function requestSiteUnblock( { dispatch }, action ) {
+	dispatch(
+		http(
+			{
+				method: 'POST',
+				apiVersion: '1.1',
+				path: `/me/block/sites/${ action.payload.siteId }/delete`,
+				body: {}, // have to have an empty body to make wpcom-http happy
+			},
+			action
+		)
+	);
+}
+
+export function receiveSiteUnblock( store, action, response ) {
+	// validate that it worked
+	const isUnblocked = !! ( response && response.success );
+	if ( ! isUnblocked ) {
+		receiveSiteUnblockError( store, action );
+		return;
+	}
+
+	store.dispatch(
+		plainNotice( translate( 'The site has been successfully unblocked.' ), {
+			duration: 5000,
+		} )
+	);
+}
+
+export function receiveSiteUnblockError( { dispatch }, { payload: { siteId } } ) {
+	dispatch( errorNotice( translate( 'Sorry, we had a problem unblocking that site.' ) ) );
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Sorry, there was a problem unblocking that site.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).